### PR TITLE
feat(package): add repository information to package.json

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
Signed-off-by: Aniruddha Shriwant <aniruddhashriwant@gmail.com>

# Issue #[341](https://github.com/accordproject/techdocs/issues/341)
<OVERALL SUMMARY OF PULL REQUEST>

### Changes

- Added Package Repository Information (directory) in `package.json`<INSERT DESCRIPTION OF CHANGES>

### CODE REVIEW

**Comment by irmerk** [comment]
Please don't merge until all related PRs are approved (tracked in https://github.com/accordproject/techdocs/issues/341)

**Review by dselman — APPROVED** [review]
LGTM

**Pull request merged**
